src/gtk/toga_gtk/widgets/internal/buttons/refresh.py

```diff
@@ -0,0 +1,190 @@
+from toga_gtk.libs import Gtk
+from .base import ParentPosition
+
+
+class RefreshButtonWidget(Gtk.Revealer):
+    def __init__(self, on_refresh: callable, on_close: callable, 
+                 position: Gtk.Align, margin: int, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.parent = None
+
+        self._refresh_btn = Gtk.Button.new_from_icon_name(
+            "view-refresh-symbolic", Gtk.IconSize.BUTTON)
+        self._refresh_btn.set_can_focus(False)
+        refresh_btn_context = self._refresh_btn.get_style_context()
+        refresh_btn_context.add_class("osd")
```

Comment:
"osd" isn't a very meaningful class name; is there an internal GTK reason for using this name, or could we pick something more verbose?

Reply:
It's a Gtk class. We need to remove it for the refresh and scroll button to look the way they do. Otherwise they retain their 3d-ness.